src/intro/hello-world.md

```diff
@@ -91,7 +93,7 @@ This should output to `{YourCrate}/target/debug/` at least one variation of a co
 
 ```admonish tip
 For users who want to follow bleeding-edge development, you can directly link the github repo in your `Cargo.toml` dependencies section by replacing:
-`godot = "0.1"`
+`godot = "X.X.X"`
```

Comment:
```suggestion
`godot = "0.x.y"`
```
So it's clear that the parts are different.

I think it's fair to update this if we get to version 1 😉